src/components/WrappedActionMenu/RewardsTab/RewardsLine/RewardsLine.stories.tsx

```diff
@@ -5,10 +5,14 @@ import { RewardsLine } from '@components/WrappedActionMenu/RewardsTab/RewardsLin
 import Rewards1 from '@static/svg/rewards1.svg'
 import BN from 'bn.js'
 
+import { Provider } from 'react-redux'
+import { store } from '../../../../store/index'
```

Comment:
You can add new path in tsconfig.paths.json and rename import path to '@store/index'

Reply:
renaming path will be enough, adding new to tsconfig is not necessary, it's already there